### PR TITLE
Add deploy script

### PR DESCRIPTION
**Pull request by GitThomas**
Please accept my changes!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
```
Comment (karlhorky) [inline package.json]:
```suggestion
    "eject": "react-scripts ejects",
```

File: package.json [hunk 2]
```diff
@@ -12,7 +12,9 @@
...
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```
Comment (karlhorky) [inline package.json]: This looks great!

File: package.json (hunk 1 above)
Comment (GitThomas) [reply package.json]: Lets not do this.

Comment (karlhorky) [reply package.json]: Oh right, that's not correct after all.

**Pull request merged**